src/Resources/views/bootstrap_4_layout.html.twig

```diff
@@ -7,7 +7,7 @@
             {% set locale = translationsFields.vars.name %}
 
             <li class="nav-item">
-                <a href="#{{ translationsFields.vars.id }}_a2lix_translations-fields" class="nav-link {% if app.request.locale == locale %}active{% endif %}" data-toggle="tab" role="tab">
+                <a href="#{{ translationsFields.vars.id }}_{{ fieldsNames|join('_') }}_a2lix_translations-fields" class="nav-link {% if app.request.locale == locale %}active{% endif %}" data-toggle="tab" role="tab">
```

Comment:
This should not be here — the `fieldsNames` variable does not exist in the `bootstrap_*` templates.

In fact the `bootstrap_*` templates should not change; New `macros_bootstrap_*` templates must be created (following the `macros_bootstrap_3.html.twig` one) and that's where this part should be added.

Reply:
@tchapi Sorry for this mistake, hope this time it's ok !